### ISSUE TITLE
Training a model for rotations

### ISSUE THREAD

**Issue by devarshi16** [herobd/Visual-Template-Free-Form-Parsing#4]
I turned the rotation parameters in the config to true, but the test results are not good even on an overfit dataset. Can you please point to where I can make the appropriate changes, improvements, in the code so as to improve the rotation results?

**Comment by herobd** [comment]
Which rotation parameter? The primary one in the code is to turn on handling rotated bounding boxes (as opposed to axis aligned), but this is not implemented (so I'd be surprised if it ran).

**Comment by devarshi16** [comment]
Yes, I figured that later on. I added rotational transformations directly to my dataset later on, but ended up using regular bounding boxes followed up by some image processing.  
Great work with the paper and the implementation by the way, was a lot of help.
